### PR TITLE
Remove path parameter from `POST /accounts`

### PR DESCRIPTION
**Pull request by jophish**
From the discussion https://github.com/fiatconnect/specification/pull/40#discussion_r868762032, it doesn't make sense to include parameters in both the path and the body for `POST /accounts`. This PR updates the spec to require all arguments for adding a new fiat account to be included in the request body.

Since there's now no risk for `POST /accounts` and `DELETE /accounts` to share a path, I've removed the `/delete` substring from `DELETE /accounts/delete/:fiatAccountId`.

### CODE REVIEW

**Review by lisaqyli — APPROVED** [review]

File: fiatconnect-api.md [hunk 1]
```diff
@@ -121,7 +121,7 @@
           - [3.4.2.3.3.2. Failure](#342332-failure)
           - [3.4.2.3.3.2.1. `ResourceNotFound`](#3423321-resourcenotfound)
     - [3.4.3. Fiat Account Endpoints](#343-fiat-account-endpoints)
-      - [3.4.3.1. `POST /accounts/:fiatAccountSchema`](#3431-post-accountsfiataccountschema)
+      - [3.4.3.1. `POST /accounts`](#3431-post-accounts)
         - [3.4.3.1.1. Parameters](#34311-parameters)
           - [3.4.3.1.1.1. Path Parameters](#343111-path-parameters)
           - [3.4.3.1.1.2. Request Body](#343112-request-body)
```
Comment (satish-ravi) [inline fiatconnect-api.md:127]:
```suggestion
          - [3.4.3.1.1.1. Request Body](#343112-request-body)
```

Comment (jophish) [reply fiatconnect-api.md]: ty!

**Review by satish-ravi — APPROVED** [review]

**Review by satish-ravi — COMMENTED** [review]
Actually should this also update the swagger to update `POST /accounts` and `DETELE /accounts`? Or would that be part of the batched update ticket?

**Comment by jophish** [comment]
@sravi I believe `DELETE /accounts/delete/:fiatAccountId` was already updated by @MuckT in the swagger spec. I've updated the swagger spec to match the changes in this PR, however.

**Comment by jophish** [comment]
@satish-ravi Ignore my previous comment -- there's no no risk for ambiguity between `POST` and `DELETE` for accounts, so I've removed the redundant `delete` string from `DELETE /accounts/delete/:fiatAccountId`!

**Review by satish-ravi — APPROVED** [review]

**Review by MuckT — APPROVED** [review]
🚀

**Pull request merged**